### PR TITLE
Update ScrollBar thumb template to use ThemeResource instead of StaticResource

### PR DESCRIPTION
**Pull request by StephenLPeters**
This Resource isn't overrideable in its current state and this change allows terminal to change the width of their scrollbars https://github.com/microsoft/terminal/issues/9218

### CODE REVIEW

File: dev/CommonStyles/ScrollBar_themeresources.xaml [hunk 1]
```diff
@@ -743,7 +743,7 @@
                                    Background="{ThemeResource ScrollBarPanningThumbBackground}"
                                    BorderBrush="{ThemeResource ScrollBarThumbBorderBrush}"
                                    Template="{StaticResource HorizontalThumbTemplate}"
-                                   Height="{StaticResource ScrollBarHorizontalThumbMinHeight}"
+                                   Height="{ThemeResource ScrollBarHorizontalThumbMinHeight}"
                                    MinWidth="{StaticResource ScrollBarHorizontalThumbMinWidth}"
```
Comment (RBrid) [inline dev/CommonStyles/ScrollBar_themeresources.xaml:747]: How about changing this to ThemeResource as well?

File: dev/CommonStyles/ScrollBar_themeresources.xaml [hunk 2]
```diff
@@ -782,7 +782,7 @@
                                    Background="{ThemeResource ScrollBarPanningThumbBackground}"
                                    BorderBrush="{ThemeResource ScrollBarThumbBorderBrush}"
                                    Template="{StaticResource VerticalThumbTemplate}"
-                                   Width="{StaticResource ScrollBarVerticalThumbMinWidth}"
+                                   Width="{ThemeResource ScrollBarVerticalThumbMinWidth}"
                                    MinHeight="{StaticResource ScrollBarVerticalThumbMinHeight}"
```
Comment (RBrid) [inline dev/CommonStyles/ScrollBar_themeresources.xaml:786]: How about changing this to ThemeResource as well?

**Review by bpulliam — APPROVED** [review]

**Pull request merged**